geode-core/src/main/java/org/apache/geode/internal/datasource/GemFireBasicDataSource.java

```diff
@@ -89,10 +89,24 @@ public Connection getConnection() throws SQLException {
           loadDriver();
       }
     }
+
     if (url != null) {
       Properties props = new Properties();
-      props.put("user", user);
-      props.put("password", password);
+
+      // If no default username or password is specified don't add these properties - the user may
+      // be
```

Comment:
It would be nice to fix these comments so we don't have lines with a single word on them